conf/application.conf

```diff
@@ -83,14 +83,20 @@ microservice {
     enrolment-store-proxy {
       protocol = http
       host = localhost
-      port = 9595
+      port = 7775
     }
 
     tax-enrolments {
       protocol = http
       host = localhost
       port = 9995
     }
+
+    tax-enrolment-assignment-frontend {
+      protocol = http
+      host = localhost
+      port = 7750
```

Comment:
Not sure if I'm missing context on this but:
- It looks like this is being prefixed to the ES8 call, but shouldn't it be using the enrolment store base url?
- Are these config values being overwritten in the environments?

Reply:
I read somewhere but for the life of me can't find it now that  the ES8 call should go through tax-enrolments not directly to them. Also reading the github for tax-enrolments it adds the enrolment to the session https://github.com/hmrc/tax-enrolments

Reply:
But this bit of config is for `tax-enrolment-assignment-frontend` not `tax-enrolments`? Is this just an incorrect name then?

Reply:
Sorry it was the fact you referred to the ES8 call. Yes this needs to be removed. It was put there temporarily on the ES0 call when we was having nightmares trying to get the journey tests working but now is not needed.